tests/integration/server.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/python
+
+from google.cloud import logging as gcloud_logging
+from google.cloud import monitoring as gcloud_monitoring
+from google.cloud.monitoring import MetricKind, ValueType
+from google.cloud.exceptions import Forbidden as ForbiddenException
+from google.cloud.exceptions import NotFound as NotFoundException
+from oauth2client.client import GoogleCredentials
+
+import logging
+import time
+
+from flask import Flask, request, jsonify
+app = Flask(__name__)
+
+
+@app.route('/')
+def hello_world():
+	return 'Hello World!'
+
+
+@app.route('/logging', methods=['POST'])
+def _logging():
+	request_data = request.get_json()
+	if request_data is None:
+		raise ErrorResponse("Unable to parse request JSON: did you set the Content-type header?")
+	log_name = request_data.get('log_name', '')
+	if log_name == '':
+		raise ErrorResponse("please provide log name")
+	token = request_data.get('token', '')
+	if token == '':
+		raise ErrorResponse("please provide token name")
+
+	_log("log name is {0}, token is {1}".format(log_name, token))
+	_log(token, log_name)
+
+	return ('OK', 200)
+
+
+# TODO (nkubala): just as a note, currently the client logging API is broken
```

Comment:
there's actually a bug in the client libraries which is being addressed soon: https://github.com/GoogleCloudPlatform/google-cloud-python/issues/2572. writing the logs is easy, but reading them through the API is currently not possible because of this issue.